common/web/input-processor/test.sh

```diff
@@ -1,18 +1,19 @@
 #!/bin/bash
 
-# We should work within the script's directory, not the one we were called in.
-cd $(dirname "$BASH_SOURCE")
-
-WORKING_DIRECTORY=`pwd`
+set -eu
 
 # Include useful testing resource functions
 ## START STANDARD BUILD SCRIPT INCLUDE
 # adjust relative paths as necessary
 THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
-. "$(dirname "$THIS_SCRIPT")/../../../../resources/build/build-utils.sh"
-. "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
+. "$(dirname "$THIS_SCRIPT")/../../../resources/build/build-utils.sh"
 ## END STANDARD BUILD SCRIPT INCLUDE
 
+. "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
```

Comment:
... is this line _not_ part of the standard build script include?  It's used so often, it may as well be.

Reply:
The standard build script include is just build-utils.sh. I want to use that pattern so that if/when we update it in the future, it's a clean update for review. I am trying to move other things out of the `## STANDARD BUILD SCRIPT INCLUDE` section for consistency.

Furthermore, shellHelperFunctions.sh is not used everywhere. I also plan to eliminate it eventually as it (a) overlaps build-utils.sh (e.g. `die()` vs `fail()`) and (b) is just a set of functions which could be put into build-utils.sh anyway with very little risk.